src/mock/quic_p2p/mod.rs

```diff
@@ -82,8 +82,8 @@ impl QuicP2p {
     /// If the peer is not connected, it will attempt to connect to it first
     /// and then send the message. This can be called multiple times while the peer is still being
     /// connected to - all the sends will be buffered until the peer is connected to.
-    pub fn send(&mut self, peer: Peer, msg: NetworkBytes, msg_id: u64) {
-        self.inner.borrow_mut().send(peer.peer_addr(), msg, msg_id)
+    pub fn send(&mut self, peer: Peer, msg: NetworkBytes, token: u64) {
```

Comment:
I think it would be a good idea to make the mock methods accept a value of type `quic_p2p::Token`, too - this way we'll remain compatible if the type changes within quic-p2p.
(This applies to all the other occurences of a `token` parameter/value, too - I won't add comments there to limit spam.)